### ISSUE TITLE
Autocomplete suggestions not read in Chrome

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#17360]
### Steps to reproduce:
1. Go to the GMail sign in page
2. Start typing an address you have used before so it offers a drop down list of suggestions
3. down arrow through the suggestions

### Actual behavior:

At Step 3 NVDA either reads "blank" or whatever text was typed into the edit field already.

### Expected behavior:

It should report the suggestion which currently has focus.

#### NVDA version:

NVDA 2024.4 installed

#### Windows version:

Windows 11 (64-bit) Version: 23H2, Build: 22631.4391 

#### Name and version of other software in use when reproducing the issue:

Chrome Version: 130.0.6723.92 

#### Does the issue still occur after restarting your computer?

Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

Tried back to NVDA 2021.3.5.  Also tried 2024.4 with UIA in Chromium set to various values without any difference.

#### If NVDA add-ons are disabled, is your problem still occurring?

No add-ons.

**Comment by douglawlor** [comment]
I can also confirm the same thing is happening on Edge. 

System configuration
NVDA installed/portable/running from source:
NVDA version:
NVDA 2024.4 installed
Windows version:
Edition	Windows 11 Home
Version	24H2 Installed on	‎10/‎11/‎2024 OS build 26120.2200 Experience	Windows Feature Experience Pack 1000.26100.33.0
Name and version of other software in use when reproducing the issue:
Microsoft Edge Version 131.0.2903.14 (Official build) stable app, beta channel (64-bit)
Chrome Version 130.0.6723.92 (Official Build) (64-bit)
Other information about your system:
Other questions
Does the issue still occur after restarting your computer?
Yes
Have you tried any other versions of NVDA? If so, please report their behaviors.
If NVDA add-ons are disabled, is your problem still occurring?
yes
Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by tmthywynn8** [comment]
Very odd. using [GitHub's sign in page](https://github.com/login), Microsoft Edge seems to read the Autofill dialog without issue. However with Google Chrome, the choice autopopulates, but if you clear Chrome's fill, I can confirm NVDA also says "blank". Narrator does read Chrome's autofill implementation though with some success.

* Microsoft Windows 10 Enterprise Version 21H2 (OS Build 19044.3086)
* Microsoft Edge for Business Version 130.0.2849.68 (Official build) (64-bit)
* Google Chrome Version 130.0.6723.92 (Official Build) (64-bit)
* NVDA 2024.4 installed; 2024.3 temp copy as ran from installer.

**Comment by Adriani90** [comment]
I cannot reproduce this with NVDA 2024.4.2 on Chrome a34 or Edge 131. When I enter the first letter of my name in the email address field, NVDA reports that a list of suggestions opens and when I press up and down arrow key I can navigate the list.

Are you still having this issue?

**Comment by seanbudd** [comment]
Hi @douglawlor @tmthywynn8 @Qchristensen - is this still an issue you can reproduce with the latest NVDA, Chrome and Edge?

**Comment by seanbudd** [comment]
Please let us know if this is still an issue and we will reopen it

**Comment by Qchristensen** [comment]
I just tried with:
NVDA 2024.4.2
Chrome Version: 132.0.6834.197 
Windows 11 (64-bit) Version: 24H2, Build: 26100.3194 

And it is reading the autocomplete field now.

**Comment by douglawlor** [comment]
Autofill has been working for some time here. Doug Sent from my iPhoneOn Feb 18, 2025, at 11:33 PM, Quentin Christensen ***@***.***> wrote:﻿
  I just tried with:
NVDA 2024.4.2
Chrome Version: 132.0.6834.197
Windows 11 (64-bit) Version: 24H2, Build: 26100.3194
And it is reading the autocomplete field now.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>



Qchristensen left a comment (nvaccess/nvda#17360)
I just tried with:
NVDA 2024.4.2
Chrome Version: 132.0.6834.197
Windows 11 (64-bit) Version: 24H2, Build: 26100.3194
And it is reading the autocomplete field now.

**Comment by WestonThayer** [comment]
Perhaps this should be for a separate issue, but I think I can still reproduce the issue with github.com/login mentioned above https://github.com/nvaccess/nvda/issues/17360#issuecomment-2455025503.

I recorded NVDA 2024.4 and Chrome 132 on Windows 10 and narrated below (note GitHub mutes embedded videos by default). Chrome visibly autofills the `Username or email address` field with my GitHub username, but NVDA announces `Username or email address edit required blank` when the github.com/login page loads and the field is autofocused. NVDA+Tab to reannounce still reports `blank`. The insertion cursor is at the beginning of the field, maybe that has something to do with it.

https://github.com/user-attachments/assets/89d082b5-2ff0-4a8d-9a1f-f6c3b1751a7a

**Comment by seanbudd** [comment]
@WestonThayer - are you still experiencing this on the latest chrome and NVDA 2025.1?

**Comment by WestonThayer** [comment]
Yes, I still repro this on Chrome 137 and NVDA 2025.1beta8 with the github.com example.

**Comment by BF150** [comment]
Hi,
this is still reproducable in the newest version 2026.2 (2026.2.0.57664) using the[ MDN playgrounds](https://developer.mozilla.org/en-US/play?uuid=b61433d1e591129782703436e61bc52b5afca4cb&state=jVNdb9NAEPwrq30lHyVCFTKuBZV44AEJ0Seke1n71vHB%2Bda9W7sJVf87chw3JAXBmz0zt56bWT9io63HDPNaYluYAJBbN0DlKaUbg9yS87eyM3jgAHJPJXuoJc7sB2sjp2Sw%2BCZ9hAMGNIH5%2BiAv8jLCep7gQtfr9Azg7Is5M6X7jmfyhCb3c0Sv35yglnaew1abCzzyfe8i2%2Fm981RxI97yaL5PHAO1%2FL7kpEkpat%2FxwHFVSXua0ZEqx3BjcPXqUmnMuVad%2BtHbF8%2BUGLoog7MMEvweCG45KdxNp%2BHjwBEqiZ1EUj4PzeAxq3xt3VCY8KKVllOiLf%2B1lyNvsPjK9z0n%2FWMPyjulyHRWxfPRGa3Ej598e%2FVbrvJwgP436c97SI1wAooMKgLqto0ugIKFT9DQwKPvrahygEYeQAUshYpXBot8Pfs8y2Teo3lLUl%2B2Tg3CQL4fgTsOFo7XPyaar49LjgusUsIMS7F7eByn1RI0g9fX3Q4ShbRMHF39zoSnMf7V%2FBtM2o6sdWG7LEVV2gw2V93uWXrq5t%2FiqbODzLvAy4bHYDLYbC40WUa1cpyklQTl0a3BzOCkwwV%2BHy%2BEC9SGW8YMLcUf%2BPQL&srcPrefix=%2Fen-US%2Fdocs%2FWeb%2FHTML%2FReference%2FElements%2Finput%2Femail%2F). 

I used Edge here:
<img width="1875" height="624" alt="Image" src="https://github.com/user-attachments/assets/9549152d-289d-468a-bf79-e1a7f92ed625" />

1. navigate to the email input 
2. auto fill opens
3. arrow down sets the focus on "test@test.test" 
4. NVDA says "Leer" (or whatever yout language has for blank :) )
5. next arrow down reads "Persönliche  Informationen verwalten" as intended
6. arrow up afterwards reads "test@test.test" 

<img width="1080" height="345" alt="Image" src="https://github.com/user-attachments/assets/f59761e5-1ba2-4748-8eb9-b1178d306c6a" />
